libraries/Microsoft.Bot.Builder/BotAdapter.cs

```diff
@@ -287,20 +287,6 @@ protected async Task RunPipelineAsync(ITurnContext turnContext, BotCallbackHandl
             // Call any registered Middleware Components looking for ReceiveActivityAsync()
             if (turnContext.Activity != null)
             {
-                if (turnContext.Activity.Locale != null)
```

Comment:
As much as I'd love to remove this, it would be a breaking change in that users may expect the turn.locale to be set here so we cannot make the decision of removing this lightly

Reply:
So I'll undo this file change